cdap-docs/admin-manual/source/operations/logback.rst

```diff
@@ -0,0 +1,12 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+======================
+Master Service Logback
+======================
+
+:term:`Master Services <master services>` use ``logback-container.xml``, packaged with the CDAP distribution,
+for logging configuration. This ``logback`` does log rotation once a day at midnight and expires logs older than
+14 days. Changes can be made to ``logback-container.xml`` but the ``cdap-master`` process will need to be restarted
```

Comment:
14 days. Changes can be made to ``logback-container.xml``, and the ``cdap-master`` process...